_includes/head.html

```diff
@@ -13,4 +13,8 @@
   {% if jekyll.environment == 'production' and site.google_analytics %}
   {% include google-analytics.html %}
   {% endif %}
+
+  {% if site.gems contains 'jekyll-seo-tag' %}
+  {% include seo.html %}
```

Comment:
That won't work because liquid doesn't lazy evaluate, the include is to lazily evaluate only, and only if the tag is present.

Otherwise, you will get an undefined tag error.